schemas/Interaction.json

```diff
@@ -82,7 +82,9 @@
         {"const": "MobileAllChatsFilterFavourites", "description": "User tapped the Favourites filter in the All Chats filter tab."},
         {"const": "MobileAllChatsFilterUnreads", "description": "User tapped the Unreads filter in the All Chats filter tab."},
         {"const": "MobileAllChatsFilterPeople", "description": "User tapped the People filter in the All Chats filter tab."},
-        {"const": "MobileSpaceCreationValidated", "description": "User validated the creation of a new space."}
+        {"const": "MobileSpaceCreationValidated", "description": "User validated the creation of a new space."},
+        {"const": "MobileRoomListRoomContextMenuUnreadToggle", "description": "User adjusted their unread rooms using the context menu on a room in the room list"},
+        {"const": "MobileRoomListRoomContextMenuFavouriteToggle", "description": "User adjusted their favourites using the context menu on a room in the room list"}
```

Comment:
I think @ganfra reported that the user can toggle the favorite from the room detail as well. Do we want to create Event for this too?
Also, do we want to distinguish "set favorite" and "unset favorite"? It will not be the case here IIUC.

Last point, sentences of description should end with periods. Element Android quality check will complain. It's OK for Element X Android, which uses this as a library.

Reply:
Okay, then I think I'll take out favorites from this PR to not complicate things. Will add the full stops 👍

Reply:
Alrighty, updated 👍